src/Database/LSMTree/Normal.hs

```diff
@@ -32,13 +32,13 @@ module Database.LSMTree.Normal (
   , Common.MergeTrace (..)
 
     -- * Table sessions
-  , Session
+  , Session (..)
   , withSession
   , openSession
   , closeSession
 
     -- * Table handles
-  , TableHandle
```

Comment:
I'd prefer if we only export representations via an exposed `.Internal` module, so they're not part of the public API.